<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="en">
  <context>
    <name>QuickTaskDialog</name>
    <message>
      <location filename="..\..\ui\quick_task_dialog.py" line="42" />
      <source>Szybkie dodawanie zadania</source>
      <translation>Quick Add Task</translation>
    </message>
    <message>
      <location filename="..\..\ui\quick_task_dialog.py" line="62" />
      <source>Zadanie:</source>
      <translation>Task:</translation>
    </message>
    <message>
      <location filename="..\..\ui\quick_task_dialog.py" line="67" />
      <source>Wpisz treść zadania...</source>
      <translation>Enter task content...</translation>
    </message>
    <message>
      <location filename="..\..\ui\quick_task_dialog.py" line="84" />
      <source>Dodaj zadanie</source>
      <translation>Add Task</translation>
    </message>
    <message>
      <location filename="..\..\ui\quick_task_dialog.py" line="406" />
      <location filename="..\..\ui\quick_task_dialog.py" line="89" />
      <source>Anuluj</source>
      <translation>Cancel</translation>
    </message>
    <message>
      <location filename="..\..\ui\quick_task_dialog.py" line="177" />
      <source>Wpisz</source>
      <translation>Enter</translation>
    </message>
    <message>
      <location filename="..\..\ui\quick_task_dialog.py" line="210" />
      <source>Brak opcji</source>
      <translation>No options</translation>
    </message>
    <message>
      <location filename="..\..\ui\quick_task_dialog.py" line="213" />
      <source>Błąd ładowania</source>
      <translation>Loading error</translation>
    </message>
    <message>
      <location filename="..\..\ui\quick_task_dialog.py" line="215" />
      <source>Brak konfiguracji</source>
      <translation>No configuration</translation>
    </message>
    <message>
      <location filename="..\..\ui\quick_task_dialog.py" line="288" />
      <location filename="..\..\ui\quick_task_dialog.py" line="280" />
      <location filename="..\..\ui\quick_task_dialog.py" line="226" />
      <source>Błąd</source>
      <translation>Error</translation>
    </message>
    <message>
      <location filename="..\..\ui\quick_task_dialog.py" line="226" />
      <source>Wpisz treść zadania!</source>
      <translation>Enter task content!</translation>
    </message>
    <message>
      <location filename="..\..\ui\quick_task_dialog.py" line="274" />
      <source>Sukces</source>
      <translation>Success</translation>
    </message>
    <message>
      <location filename="..\..\ui\quick_task_dialog.py" line="274" />
      <source>Zadanie zostało dodane!</source>
      <translation>Task has been added!</translation>
    </message>
    <message>
      <location filename="..\..\ui\quick_task_dialog.py" line="280" />
      <source>Nie udało się dodać zadania do bazy danych!</source>
      <translation>Failed to add task to database!</translation>
    </message>
    <message>
      <location filename="..\..\ui\quick_task_dialog.py" line="288" />
      <source>Wystąpił błąd podczas dodawania zadania:</source>
      <translation>An error occurred while adding the task:</translation>
    </message>
  </context>
</TS>
